code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -23,7 +23,7 @@
 			. = 1
 
 /mob/living/carbon/human/mob_negates_gravity()
-	return ((shoes && shoes.negates_gravity()) || dna.species.negates_gravity(src))
+	return ((shoes && shoes.negates_gravity()) || (dna && dna.species && dna.species.negates_gravity(src)))

```

Comment:
these safety checks are not needed, if the mob doesnt have `dna` or `species` something is really wrong